RewiMod-Forge1.9/src/main/java/tv/rewinside/rewimod/forge/gui/objects/GuiRewiConnectButton.java

```diff
@@ -33,11 +38,25 @@ public GuiRewiConnectButton(int buttonID, int x, int y) {
 	@Override
 	public void drawButton(Minecraft mc, int mouseX, int mouseY) {
 		CoreGuiDrawer.drawButtonRewiConnect(this, this.visible, this.xPosition, this.yPosition, mouseX, mouseY, this.width, this.height);
+
+		if (CoordinateUtil.inbetween(xPosition, yPosition, mouseX, mouseY, width, height)) {
+			this.drawString(mc.fontRendererObj, WordUtils.capitalizeFully(ConnectType.getConnectType().name().toLowerCase()),
+					this.xPosition + 22, this.yPosition + 5, Color.WHITE.getRGB());
+		}
 	}
 
 	@Override
-	public void onClick() {
-		CoreGuiActions.clickButtonRewiConnect();
+	public void onClick(int mouseButton) {
+		if (mouseButton == 0) {
```

Comment:
das gesamte Statement kann in die RewiMod-Core rein. Somit haben wir weniger doppelten code